tailwind.config.js

```diff
@@ -0,0 +1,61 @@
+const variableTokens = require('./lib/tokens/json/css-variables-nested.json');
+const staticTokens = require('./lib/tokens/json/variables-nested.json');
+
+module.exports = {
+  /**
+   * The main value in TW utility classes is for Storybook stories & recipes.
+   * We avoid using them in component styles to reduce chance of conflict with other libraries.
+   * Please configure downstream Tailwind config purge to include app files if necessary.
+   */
+  content: [
+    './src/components/**/*.stories.{ts,tsx}',
+    './src/components/**/*Example.tsx',
+    './.storybook/**/*.{js,jsx,ts,tsx}',
+  ],
+  corePlugins: {
+    /**
+     * Disable Tailwind's preflight (CSS reset) styles.
+     * We're only using Tailwind for stories/recipes, and some of those
+     * (esp. documentation pages) depend on browser default styles.
+     *
+     * Remove this setting if we are turning on Tailwind for component styles,
+     * so we will better match consuming apps.
+     */
+    preflight: false,
```

Comment:
Unfortunately, it seems some of our stories depend on browser default styles 🤔  See https://www.chromatic.com/build?appId=61313967cde49b003ae2a860&number=2504. It's not the biggest deal to have preflight turned off because
1. It preserves how the stories look rn
2. Technically, having preflight on would make our story environment closer to traject. But traject has its own bootstrap nonsense that affects how things look there anyways

It'd be good to eventually turn it on / work through the needed story changes, but not high pri

Reply:
Created [backlog ticket](https://app.shortcut.com/czi-edu/story/213428) for preflight/browser default styles